lib/CMakeLists.txt

```diff
@@ -85,13 +86,21 @@ if(VGG_VAR_TARGET MATCHES "^iOS")
     file(RENAME ${NODEJS_MOBILE_DIR} "${CMAKE_CURRENT_SOURCE_DIR}/${NODE_FOLDER}")
   endif()
 
-  # Download the pre-built binary (NodeMobile.xcframework)
-  set(LIB_NODE_FOLDER "nodejs-mobile-ios" CACHE STRING "lib node folder name" FORCE)
+  # Download the iOS pre-built binary (NodeMobile.xcframework)
+  set(LIB_NODE_FOLDER "nodejs-mobile-lib" CACHE STRING "lib node folder name" FORCE)
```

Comment:
Please rename the folder name `nodejs-mobile-ios` in `src/Entry/Container/CMakeLists.txt` as well